### ISSUE TITLE
baseline feature performance issue

### ISSUE THREAD

**Issue by toco2270853** [andi611/Self-Supervised-Speech-Pretraining-and-Representation-Learning#36]
Hi, thanks for the great code, really appreciate it!

When I'm running downstream evaluation written by my self, I found that mfcc feature with utterance based speaker classification can reach high accuracy. But when I use s3prl to run. I get poor result, with command:
1. python preprocess/preprocess_libri.py --feature_type=mfcc --delta=True --delta_delta=True
2. python run_downstream.py --run=speaker_utterance --upstream=baseline --config=config/downstream.yaml --input_dim=39
the accuracy is 0.03~0.05

And I made some modification:
1. change learning rate in downstream.yaml to  0.001
2. python preprocess/preprocess_libri.py --feature_type=mfcc --delta=True --delta_delta=True --apply_cmvn=False
3. python run_downstream.py --run=speaker_utterance --upstream=baseline --config=config/downstream.yaml --input_dim=39

and the accuracy went to 0.88, could you please check for this? Did I misunderstood anything?

**Comment by andi611** [comment]
> 2. python preprocess/preprocess_libri.py --feature_type=mfcc --delta=True --delta_delta=True --apply_cmvn=False

Hi, thanks for the feedback!
I think the major cause is `--apply_cmvn=False`.
The normalization of cmvn will illuminate global information (a.k.a speaker identity)

**Comment by toco2270853** [comment]
Get it ! thanks for the quick reply!
